activiti-cloud-acceptance-scenarios/runtime-acceptance-tests/src/main/java/org/activiti/cloud/qa/story/Tasks.java

```diff
@@ -386,19 +387,24 @@ public void setTaskVariables() {
 
     @Then("task variables are visible in rb and query")
     public void checkTaskVariablesAreTheSameInRBAndQuery() {
-        Map<String, Object> generatedMapRuntime = new HashMap<>();
-        Map<String, Object> generatedMapQuery = new HashMap<>();
+        await()
+            .pollInterval(Duration.ofMillis(250L))
+            .pollInSameThread()
+            .untilAsserted(() -> {
```

Comment:
is there a default timeout where we stop waiting in case the assertion never gets be valid please?

Reply:
Yes, 10 seconds. Since the test passes even with no timeout, I assumed the default is enough for a first try.

Reply:
sounds good, thanks :+1: